torchvision/csrc/io/image/cpu/readjpeg_cpu.cpp

```diff
@@ -117,7 +117,7 @@ torch::Tensor decodeJPEG(const torch::Tensor& data, ImageReadMode mode) {
        */
       default:
         jpeg_destroy_decompress(&cinfo);
-        TORCH_CHECK(false, "Provided mode not supported");
+        TORCH_CHECK(false, "Provided mode not supported as the input is JPEG");
```

Comment:
This might read more naturally to native english speakers. What you think?
```suggestion
        TORCH_CHECK(false, "The provided mode is not supported for JPEG files");
```

Reply:
Updated. Thanks!